tools/clang/unittests/HLSL/CMakeLists.txt

```diff
@@ -76,6 +75,7 @@ set(HLSL_IGNORE_SOURCES
   )
 
 add_clang_unittest(clang-hlsl-tests
+  AllocatorTest.cpp
```

Comment:
Can we also add `add_test()` to `clang-hlsl-tests` so that it can be invoked by `ctest`? Like

https://github.com/Microsoft/DirectXShaderCompiler/blob/3987db0fa83c9233b93af7a8db8a98feaaa9f0e2/tools/clang/unittests/SPIRV/CMakeLists.txt#L44-L46